src/index.html

```diff
@@ -1,13 +1,58 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+
+  <body>
+    <div class="card" data-qa="card">
+      <img
+      class="card__image"
+      src="./images/imac.jpeg"
+      alt="TV APPLE A1419 image
+      ">
```

Comment:
check the tabulation
```suggestion
      <img
        class="card__image"
        src="./images/imac.jpeg"
        alt="TV APPLE A1419 image"
      >
```